lib/util/periods.js

```diff
@@ -852,7 +852,10 @@ shaka.util.PeriodCombiner = class {
     if (output.originalId == null) {
       output.originalId = input.originalId;
     } else {
-      output.originalId += ',' + (input.originalId || '');
+      const newOriginalId = (input.originalId || '');
+      if (newOriginalId && !output.originalId.endsWith(newOriginalId)) {
+        output.originalId += ',' + newOriginalId;
+      }
```

Comment:
In one of test cases we have resulting id `stream1,stream2,stream1,stream2`. Do we want to address such a case as well?

Reply:
I don't have any streams with that case, I don't know if it's a real case. If it causes problems in the future, I would address it.